i18n/en.json

```diff
@@ -824,6 +824,14 @@
     "translation": "shrug"
   },
   {
+    "id": "api.command_help.desc",
+    "translation": "Open the Mattermost help page"
```

Comment:
Will this command open the help page or send the link to the user? and then the user open the link?